.github/workflows/nix-parse.yml

```diff
@@ -0,0 +1,40 @@
+name: "Check whether nix files are parseable"
+
+permissions: read-all
+
+on:
+  # avoids approving first time contributors
+  pull_request_target:
+    branches-ignore:
+      - 'release-**'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    if: "github.repository_owner == 'NixOS' && !contains(github.event.pull_request.title, '[skip treewide]')"
+    steps:
+    - name: Get list of changed files from PR
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        gh api \
+          repos/NixOS/nixpkgs/pulls/${{github.event.number}}/files --paginate \
+          | jq '.[] | select(.status != "removed" and (.filename | endswith(".nix"))) | .filename' \
```

Comment:
Should we get raw strings from jq with `--raw-output` to remove the JSON quotes surrounding the file paths?

Reply:
I see you've got [a pull request](https://github.com/NixOS/nixpkgs/pull/281366). Thanks!